### ISSUE TITLE
[Unblock Legitimate Site] https://decentraland.org

### ISSUE THREAD

**Issue by nachomazzara** [MetaMask/eth-phishing-detect#14607]
**Domain:** https://decentraland.org

**Details:** Decentraland is a 3D virtual world browser-based platform. Users may buy virtual plots of land in the platform as NFTs via the MANA cryptocurrency, which uses the Ethereum blockchain. Designers can create and sell clothes and accessories for the avatars to be used in the virtual world. We offer transaction and meta transaction. The latter is being marketed as a potential risk, which is not true.

![image](https://github.com/MetaMask/eth-phishing-detect/assets/7549152/f42f130c-95ba-4b97-b36c-f07fb95b3d2d)

- [X] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by AlexHerman1** [comment]
this warning is different from the functionality of this repo. i will flag this to the correct channel.

**Comment by Kaze-No-Kai** [comment]
@AlexHerman1 can you please share the new issue or where this was flagged so we can keep track of it? Thanks
@nachomazzara fyi
